.eslintrc.yml

```diff
@@ -45,6 +46,15 @@ rules:
   react/no-unescaped-entities: 0
   import/no-unresolved: error #re-add no-unresolved
   unicorn/custom-error-definition: error
+  inclusive-language/use-inclusive-words:
+    - error
```

Comment:
Can we start with a populated list? There are terms that we definitely should reject from the beginning. I can think of:

- master
- slave
- dummy

We probably should have a list.